Formula/libzt.rb

```diff
@@ -0,0 +1,29 @@
+class Libzt < Formula
+  desc "ZeroTier: libzt -- An encrypted P2P networking library for applications"
```

Comment:
Can remove the `libzt` from here as it's in the description already. I'd suggest removing everything before the `--`

Reply:
Removed